SingleSampleQc.wdl

```diff
@@ -24,7 +24,7 @@ version 1.0
 ## - [VerifyBamID2](https://github.com/Griffan/VerifyBamID)
 
 # Git URL import
-import "https://raw.githubusercontent.com/genome/qc-analysis-pipeline/master/tasks/Qc.wdl" as QC
+import "tasks/Qc.wdl" as QC
```

Comment:
@acoffman Do you recall why we didn't use relative imports in the past? I feel like this was a limitation of Terra/Cromwell and/or Dockstore.

Reply:
I found a thread on Dockstore that discusses when support was added to both Terra and Dockstore to accept relative paths around Mar/April 2020:
https://discuss.dockstore.org/t/relative-import-path-for-terra/3476/3

I'd say this all looks good then.

Reply:
Yeah, I switched the other branch to a relative import after they announced support for it.  424e885ce263eadba2bcea50eb8466272bf2915d was the original change, so we've been using it for awhile.